pm_kext/col/verdict_cache.c

```diff
@@ -22,6 +22,29 @@
 #include "pm_utils.h"
 #include "pm_debug.h"
 
+// Used to hold the number of modifications performed on the cache
```

Comment:
Suggestion:
`Used to hold` could mean `Previously held`. Keep it simple with `Holds ...`.

Reply:
On that topic, please use a more descriptive variable name.